src/Components/DeleteModal/DeleteModal.tsx

```diff
@@ -0,0 +1,94 @@
+/** @format */
+
+import React, { useState } from "react";
+import { deleteImg } from "@/Assets/Images";
+
+import {
+  Button,
+  Dialog,
+  DialogActions,
+  DialogContent,
+  DialogContentText,
+  DialogTitle,
+  Slide,
+} from "@mui/material";
+import { TransitionProps } from "@mui/material/transitions";
+import axios from "axios";
+import { FormControl } from "@mui/base/FormControl";
+
+const Transition = React.forwardRef(function Transition(
+  props: TransitionProps & {
+    children: React.ReactElement<any, any>;
+  },
+  ref: React.Ref<unknown>
+) {
+  return <Slide direction="up" ref={ref} {...props} />;
+});
+
+const DeleteModal = ({
+  itemId,
+  getData,
+  handleSubmit,
+  handleCloseDialog,
+  openDialog,
+}) => {
+  const deleteHandler = () => {
+    axios
+      .delete(`http://154.41.228.234:3000/api/v0/admin/rooms/${itemId}`, {
+        headers: {
+          Authorization: `Bearer eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJfaWQiOiI2NThhMTgyYjQ3ZWUyYjE0Zjk1NDY5OTAiLCJyb2xlIjoiYWRtaW4iLCJ2ZXJpZmllZCI6ZmFsc2UsImlhdCI6MTcwNDU2OTYyMywiZXhwIjoxNzA1Nzc5MjIzfQ.noHrXaGYeIqjYk4A3GhBcLS3CWTawveLCf7Iqr691AQ`,
+        },
+      })
+      .then((response) => {
+        console.log(response);

```

Comment:
We should remove logs before merging to the main branch to avoid revealing sensitive data to users.
We should also show toast messages to give a better user feedback in our application of what is going in our application whether something went well or something went wrong.